internal/api/logs/v1/http.go

```diff
@@ -0,0 +1,145 @@
+package http
```

Comment:
It looks like we need a gofmt on this file

Reply:
Ahh my goimports hit this file.